pkg/containerimagebuilderpusher/containerimagebuilder.go

```diff
@@ -16,4 +16,7 @@ type BuilderPusher interface {
 
 	// GetBaseImageRegistry returns onbuild base registry
 	GetBaseImageRegistry(registry string) string
+
+	// GetSecretName returns secret with credentials to push/pull from docker registry
+	GetSecretName() string
```

Comment:
```suggestion
	GetRegistryCredentialsSecretName() string
```

I mean, secret is the abstraction name, not the instance :)

Reply:
changed to `GetDefaultRegistryCredentialsSecretName`